airflow/contrib/utils/gcp_input_validator.py

```diff
@@ -0,0 +1,11 @@
+from airflow import AirflowException
+
+
+class GcpInputValidationMixin(object):
```

Comment:
I'd consider simplifying the name to `InputValidationMixin`.
After all it's not GCP-specific and could be used for any type of Operators.